lib/ansible/utils/module_docs_fragments/hpe3par.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+class ModuleDocFragment(object):
+
+    # HPE 3PAR doc fragment
+    DOCUMENTATION = '''
+options:
+    storage_system_ip:
+      description:
+        - "The storage system IP address."
+      required: true
+    storage_system_password:
+      description:
+        - "The storage system password."
+      required: true
+    storage_system_username:
+      description:
+        - "The storage system user name."
+      required: true
+
+requirements:
+  - "3PAR OS - 3.2.2 MU6, 3.3.1 MU1"
```

Comment:
The 3PAR firmware version check is done at the library level

Reply:
Because:
> Logout has been made idempotent in the hpe3par_sdk library now. It will not raise an exception is not already logged in as well ([here](https://github.com/ansible/ansible/pull/39656#discussion_r190213227))

and
https://github.com/ansible/ansible/blob/4e6a50cf4433ec96edc0ee7ac6d20eb0c392272c/lib/ansible/utils/module_docs_fragments/hpe3par.py#L39
then the `hpe3par_sdk` version should be checked (preferably in `module_utils.hpe3par` in order to share the check between modules).